public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
1 more question: should we also update the copyright year to 2020 in README.md? Is there any guideline on this?

Reply:
not sure.  I updated anyway.